--- a/usr/share/applications/franz.desktop	2021-02-26 10:44:16.001306978 +0100
+++ b/usr/share/applications/franz.desktop	2021-02-26 10:45:49.793773097 +0100
@@ -5,6 +5,6 @@
 Type=Application
 Icon=franz
 StartupWMClass=Franz
-Comment=Franz is your messaging app / former Emperor of Austria and combines chat & messaging services into one application. Franz currently supports Slack, WhatsApp, Gmail, Facebook Messenger, Telegram, Google Hangouts, GroupMe, Skype and many more. You can download Franz for free for Mac & Windows.
+Comment=Free messaging app for services like WhatsApp, Slack, Messenger and many more
 MimeType=x-scheme-handler/franz;
 Categories=Network;InstantMessaging;
